docs/preview/02-Guidance/use-with-dotnet-and-aspnetcore.md

```diff
@@ -4,29 +4,34 @@ layout: default
 ---
 
 # Using Arcus & Serilog in ASP.NET Core
+The Arcus Application Insights Serilog sink is a great way to simplify telemetry tracking and application logging. Unlike Microsoft's `TelemetryClient` for telemetry tracking, Arcus uses the common `ILogger` infrastructure to track and link telemetry in Application Insights.
 
-When using Arcus Observability & Serilog it is mandatory that your application is configured correctly to ensure the telemetry is correctly send to Azure Application Insights.
+This user guide will walk through the steps to configure the Arcus Application Insights Serilog sink in ASP.NET Core applications.
 
-## Setting up Serilog with ASP.NET Core
-
-Using Serilog with ASP.NET Core requires you to install an ASP.NET Core specific Serilog package:
+## Installation
+The example in this user guide uses following packages. 
 
 ```shell
 PM > Install-Package -Name Serilog.AspNetCore
+PM > Install-Package -Name Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights
+PM > Install-Package -Name Arcus.Security.Providers.AzureKeyVault
+PM > Install-Package -Name Arcus.WebApi.Logging
 ```
 
-When using Arcus.Observability, we expect you to setup Serilog as your logging infrastructure.
-We encourage you to follow the standard [Serilog instructions](https://github.com/serilog/serilog-aspnetcore) on setting up Serilog for ASP.NET Core. Make sure to call [`UseSerilog`](https://www.nuget.org/packages/Serilog.AspNetCore).
+### Serilog.AspNetCore
+Since Arcus uses Serilog, we expect you to setup Serilog as your logging infrastructure. This is also Microsoft's recommended logging system ([more info](https://serilog.net/)).
```

Comment:
Is it ? Do we have a source ?

Reply:
Yeah, I was always under impression that it was since the Microsoft docs also uses Serilog in their examples.

Reply:
I'll resolve this but we can always come back on it.